docs/source/search/features.rst

```diff
@@ -0,0 +1,77 @@
+Internal Search
+===============
+
+The Internal Search is a default SQL based engine implementation, created to support the most basic queries.
+While the performance and range of features might not be the same, depending on your instance, if your volume of
+records is small and retrieval needs are simple, this module should be enough.
+
+It supports the two types of records from a total of six targets:
+
+- Events
+- Categories
+- Contributions
+- Attachments
+- Notes
+
+.. note::
+    The Internal Search only supports text-based search on titles, description and notes content.
+
+External Search Service
+=======================
+
+Indico supports several powerful features for aggregation, filtering and retrieval of information natively
+covered by the out-of-the-box `Citadel`_ service.
+
+Aggregations
+------------
+
+Aggregations, `as seen in Elastic Search <https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations.html>`_,
```

Comment:
`Elasticsearch`; and maybe move the link to the end of the file like you did with Citadel?

Reply:
Sure, it's the only usage anyway, I had it in-line to avoid having this word associated with a specific aggregation link.

Reply:
Ah good point, it's really annoying that you cannot use different names for the link than the displayed text. MAybe better to leave it as-is...